HopsanCore/HopsanCore.pro

```diff
@@ -39,6 +39,53 @@ SOURCES += $${PWD}/dependencies/libnumhop/src/VariableStorage.cpp
 HEADERS += $${PWD}/dependencies/libnumhop/include/numhop.h
 #--------------------------------------------------------
 
+#-------------------------------------------------
+# Set sundials paths (compile in)
+system("cd dependencies/sundials-build && \
+cmake ../sundials \
+-DCMAKE_INSTALL_PREFIX=$${PWD}/../bin \
+-DCMAKE_BUILD_TYPE=RELEASE \
+-DBUILD_ARKODE=OFF \
+-DBUILD_CVODE=OFF \
+-DBUILD_CVODES=OFF \
+-DBUILD_IDA=OFF \
+-DBUILD_IDAS=OFF \
+-DBUILD_KINSOL=ON \
+-DBUILD_SHARED_LIBS=ON\
```

Comment:
Shared libs should be OFF if we build it in I think

Reply:
CMake enforces either static or shared libraries to be turned on. But it doesn't matter, since we don't actually build anything with their CMake system.